### PR TITLE
Features/popup

### PR DESCRIPTION
**Pull request by Wahaj-Ali**
in this branch
- Edited index.html to remove the projects section and implement it dynamically using JavaScript objects.
- Created a popup window for both mobile and desktop versions.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -10,4 +11,163 @@ menuIcon.addEventListener('click', () => {
...
+displayCards();
+
+// popup window
+const btn = document.getElementsByClassName('checkout-button');
+const popupWin = document.getElementById('popup-win');
+cards.forEach((card) => {
+  for (let i = 0; i < btn.length; i += 1) {
+    btn[i].addEventListener('click', () => {
+      if (i === ((card.id))) {
+        popupWin.style.display = 'block';
```
Comment (Munsa1) [inline script.js:124]: - [ ] Kindly follow the given design for the desktop as well as the the mobile version.

#### Your Desktop version
image

#### Provided desktop design
image

- KIndly ensure the contents are aligned properly, your contents are overflowing out of the main container.

File: script.js [hunk 2]
```diff
@@ -10,4 +11,163 @@ menuIcon.addEventListener('click', () => {
...
+                </ul>
+            </article>
+            <button type="submit" class="checkout-button button_interactions" id="projects-btn2" name="checkout-button"
+                >${card.btnText}</button>
+        </section>`;
+  });
+  document.querySelector('.desktop_projects').innerHTML = result;
+}
+
+displayCards();
```
Comment (Munsa1) [inline script.js:115]: - [ ] Kindly ensure that the cards are responsive, meaning that when the modal pop-up is opened in desktop view, as the screen size changes into mobile view so should the pop-up.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Wahaj-Ali, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] Follow the given design for the mobile view by implementing a visible `close` button for your design
#### You Mobile Design
image

#### Figma Mobile Design
image

- The heading in your design is not visible, kindly fix this.
- The paragraph should span the full width of the image, kindly fix this as well.
image
- the bottom buttons should also span the width of the image at the top, kindly fix this(check image below)
image
 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Wahaj-Ali) [reply script.js:124]: desktop-popup
It's working fine on my computer. It is just the same as the template.
@Munsa1

Comment (Wahaj-Ali) [reply script.js:124]: mobile-popup
@Munsa1 For the mobile popup, it is also fine. All the items are aligned properly with a proper gap between them.

**Review by ahmetbozaci — COMMENTED** [review]
### Hello @Wahaj-Ali 
### _This is an additional review as a result of the second opinion request._ 

- I can see you already added the close button and matched it with the Figma design.
image

image


___
_**Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away**_



### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️✔️

Hi @Wahaj-Ali, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [x] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**